tests/foreman/ui/test_errata.py

```diff
@@ -1454,108 +1533,91 @@ def test_positive_filtered_errata_status_installable_param(
-            }
-            host_details_values = session.host.get_details(client.hostname)
-            actual_values = {
-                key: value
-                for key, value in host_details_values['properties']['properties_table'].items()
-                if key in expected_values
-            }
-            for key in actual_values:
-                assert expected_values[key] in actual_values[key], 'Expected text not found'
-            property_value = 'Yes'
-            session.settings.update(f'name = {setting_update.name}', property_value)
-            assert client.execute(f'yum install -y {FAKE_9_YUM_OUTDATED_PACKAGES[1]}').status == 0
-            expected_values = {
-                'Status': 'Error',
-                'Errata': 'Security errata installable',
-                'Subscription': 'Fully entitled',
-            }
-            # Refresh the host page to get the new details
-            session.browser.refresh()
-            host_details_values = session.host.get_details(client.hostname)
-            actual_values = {
-                key: value
-                for key, value in host_details_values['properties']['properties_table'].items()
-                if key in expected_values
-            }
-            for key in actual_values:
-                assert expected_values[key] in actual_values[key], 'Expected text not found'
+
+    with session:
+        session.location.select(loc_name=DEFAULT_LOC)
+        property_value = 'Yes'
+        session.settings.update(f'name = {setting_update.name}', property_value)
+        expected_values = {
+            'Status': 'OK',
+            'Errata': 'All errata applied',
+            'Subscription': 'Fully entitled',
+        }
+        host_details_values = session.host.get_details(client.hostname)
+        actual_values = {
+            key: value
+            for key, value in host_details_values['properties']['properties_table'].items()
+            if key in expected_values
+        }
+        for key in actual_values:
+            assert expected_values[key] in actual_values[key], 'Expected text not found'
+        property_value = 'Yes'
+        session.settings.update(f'name = {setting_update.name}', property_value)
+        assert client.execute(f'yum install -y {FAKE_9_YUM_OUTDATED_PACKAGES[1]}').status == 0
+        expected_values = {
+            'Status': 'Error',
+            'Errata': 'Security errata installable',
+            'Subscription': 'Fully entitled',
+        }
+        # Refresh the host page to get the new details
+        session.browser.refresh()
+        host_details_values = session.host.get_details(client.hostname)
+        actual_values = {
+            key: value
+            for key, value in host_details_values['properties']['properties_table'].items()
+            if key in expected_values
+        }
+        for key in actual_values:
+            assert expected_values[key] in actual_values[key], 'Expected text not found'
```

Comment:
Since `expected_values` and `actual_values` are same length (same keys), I guess this should work
```suggestion
        assert expected_values == actual_values, 'Expected text not found'
```

Reply:
So this assertion fails due to one differing value
```
E    Differing items:
E    {'Errata': 'Security errata installable'} != {'Errata': 'Security errata installable clear'}
```

Reply:
The existing test already had this key loop assertion, so should we keep the same?

Reply:
Or we could revise our expectation if we get something unexpected but it seems to be correct anyway.
```
        expected_values = {
            'Status': 'Error',
            'Errata': 'Security errata installable clear',
            'Subscription': 'Fully entitled',
        }
```
Btw, why 'clear'??

Reply:
I am not sure about the clear, first time I am seeing it, 
will dig in to see why it's relevant